tests/test_data/addapinch.com/addapinch_1.json

```diff
@@ -65,6 +65,7 @@
     "servingSize": "1 serving"
   },
   "ratings": 5.0,
+  "ratings_count": 4164,
```

Comment:
Ah, this gives me an idea/question: there's a risk here that these ratings values could cause unnecessary churn (repeated changes) in our test data if we begin updating them more frequently.

That may be true for other fields too, but it would certainly be the case for this one, at least on popular recipes.

I wonder if the tests should assert on something like `ratings_count is greater-than-or-equal to this value` -- so that it can increase over time without us having to update the test assertion (and thus creating an unnecessary code diff) each time.

Reply:
This thread was resolved by the greater-or-equal-to check; thanks!